.gitignore

```diff
@@ -11,3 +11,4 @@ build/
 composer.phar
 .idea/
 vendor/
+public/js/CDash_*.min.js*
```

Comment:
Aside: Some schools of thought would check in this file and only have developers run gulp. There are advantages and disadvantages either way.